apps/directory/src/stores/filtersStore.ts

```diff
@@ -64,16 +66,15 @@ export const useFiltersStore = defineStore("filtersStore", () => {
       : false;
   });
 
-  function getValuePropertyForFacet(facetIdentifier) {
-    return facetDetails[facetIdentifier].filterValueAttribute;
+  function getValuePropertyForFacet(facetIdentifier: string) {
+    return facetDetails.value[facetIdentifier].filterValueAttribute;
```

Comment:
kinda surprised this worked without the '.value'

Reply:
I probably didn't...